kornia/augmentation/base.py

```diff
@@ -47,8 +48,7 @@ def __repr__(self) -> str:
     def __infer_input__(
```

Comment:
Remove this if it is not in need.

Reply:
I suggest we keep this as __infer_input__ is non trivial here https://github.com/kornia/kornia/blob/12c116fcbfb4073b1a87d5280a5d7dddc507086c/kornia/augmentation/base.py#L147.
Otherwise we would have to unpack the input inside forward, which makes it ugly IMO.

Reply:
I see. Do you think it is a good name? I am not sure if it is appropriate to use "infer" here. Feel free to rename it if you got a better idea.

Reply:
How about `unpack_input` or `unpack_input_tuple`?

Reply:
Great. I would vote for ```__unpack_input__```.